SIDFactoryII/source/runtime/execution/executionhandler.cpp

```diff
@@ -159,6 +179,10 @@ namespace Emulation
 				// Decrement the remaining number of samples
 				uiRemainingSamples -= uiSamplesToCopy;
 			}
+			if (clipping)
+			{
+				std::cout << "WARNING: Sound output is clipping! Lower Sound.Output.Gain to avoid distortion.\n";
+			}
```

Comment:
Not sure if this is great, since it's not running on the main thread and has to quit fast. Maybe make this clipping warning pullable from the execution handler, and output it from the main thread, or otherwise indicate the issue is occuring?

Reply:
Or just remove the logging and add a warning to the ini file that values above 1.0 could result in distorted sound.

Reply:
Removed the logging